runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -2573,6 +2573,37 @@ remoteCompilationEnd(
             comp->failCompilation<J9::AOTRelocationFailed>("Failed to relocate");
             }
          }
+      else
+         {
```

Comment:
This code is a copy-paste from CompilationThread.cpp. Let's create a method that is called twice in relevant places. Also, in line 2503 there is a debug object being defined but not being used anywehere.

Reply:
The debug object was spotted and removed in another PR that hasn't been merged yet.